admyral/server/endpoints/workflow_run_endpoints.py

```
@@ -15,7 +15,9 @@
 
 @router.get("/{workflow_id}", status_code=status.HTTP_200_OK)
 async def list_workflow_runs(
-    workflow_id: str, authenticated_user: AuthenticatedUser = Depends(authenticate)
+    workflow_id: str,
+    limit: int | None = 100,
+    authenticated_user: AuthenticatedUser = Depends(authenticate),
```

Ruff

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)